packages/vuetify/src/components/VCalendar/mixins/__tests__/calendar-with-intervals.spec.ts

```diff
@@ -182,7 +182,7 @@ describe('calendar-with-intervals.ts', () => {
     const scope = wrapper.vm.getSlotScope(parseTimestamp('2019-02-08'))
     delete scope.week
 
-    expect(scope).toMatchSnapshot()
+    // expect(scope).toMatchSnapshot()
```

Comment:
what's the reason for this being commented out?

Reply:
@nekosaur I just removed it, per clicker monkeys advice.